21_node_management/22_search_nodes.txt

```diff
@@ -10,17 +10,76 @@ about the Nodes which have been already validated, and are managed by Rudder.
 
 ==== Quick Search
 
-The easiest search tool is the Quick search: type in the search field the first
-letters of the 'Rudder ID', 'Reference', or 'Hostname'; choose the accurate Node in
-the autocompletion list; validate and look at the Node information. This search
-tool can be very useful to help you create a new search in the Advanced Search.
```

Comment:
This paragraph has been completely removed, but still is relevant to users with older versions. Please keep it, but move it to a dedicated sub-paragraph below.